aspnetcore/migration/30-to-31.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Migrate from ASP.NET Core 3.0 to 3.1
+author: scottaddie
+description: Learn how to migrate an ASP.NET Core 3.0 project to ASP.NET Core 3.1.
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 11/26/2019
+uid: migration/30-to-31
+---
+# Migrate from ASP.NET Core 3.0 to 3.1
+
+By [Scott Addie](https://github.com/scottaddie)
+
+This article explains how to update an existing ASP.NET Core 3.0 project to ASP.NET Core 3.1.
+
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+[!INCLUDE[](~/includes/net-core-prereqs-vs-3.1.md)]
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/net-core-prereqs-vsc-3.1.md)]
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+[!INCLUDE[](~/includes/net-core-prereqs-mac-3.1.md)]
+
+---
+
+## Update .NET Core SDK version in global.json
+
+If you rely upon a [global.json](/dotnet/core/tools/global-json) file to target a specific .NET Core SDK version, update its `version` property to the 3.1 version installed on your machine. For example:
+
+```diff
+{
+  "sdk": {
+-    "version": "3.0.101"
++    "version": "3.1.100-preview3-014645"
+  }
+}
+```
+
+## Update the target framework
+
+In your project file, update the [Target Framework Moniker (TFM)](/dotnet/standard/frameworks) to `netcoreapp3.1`:
```

Comment:
In the project file, update the [Target Framework Moniker (TFM)](/dotnet/standard/frameworks) to `netcoreapp3.1`: